docs/Create.md

```diff
@@ -0,0 +1,460 @@
+    };
+
+    return (
+        <Create mutationOptions={{ onSuccess }}>
+            <SimpleForm>
+                ...
+            </SimpleForm>
+        </Create>
+    );
+}
+```
+{% endraw %}
+
+You can do the same for error notifications, by passing a custom `onError`  callback.
+
+**Tip**: The notification message will be translated.
+
+## Prefilling the Form
+
+You sometimes need to pre-populate a record based on a *related* record. For instance, to create a comment related to an existing post. 
+
+By default, the `<Create>` view starts with an empty `record`. However, if the `location` object (injected by [react-router-dom](https://reacttraining.com/react-router/web/api/location)) contains a `record` in its `state`, the `<Create>` view uses that `record` instead of the empty object. That's how the `<CloneButton>` works under the hood.
+
+That means that if you want to create a link to a creation form, presetting *some* values, all you have to do is to set the location `state`. `react-router-dom` provides the `<Link>` component for that:
+
+{% raw %}
+```jsx
+import * as React from 'react';
+import { Datagrid } from 'react-admin';
+import { Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+const CreateRelatedCommentButton = ({ record }) => (
+    <Button
+        component={Link}
+        to={{
+            pathname: '/comments/create',
+            state: { record: { post_id: record.id } },
+        }}
+    >
+        Write a comment for that post
+    </Button>
+);
+
+export default PostList = () => (
+    <List>
+        <Datagrid>
+            ...
+            <CreateRelatedCommentButton />
+        </Datagrid>
+    </List>
+)
+```
+{% endraw %}
+
+**Tip**: To style the button with the main color from the MUI theme, use the `Link` component from the `react-admin` package rather than the one from `react-router-dom`.
+
+**Tip**: The `<Create>` component also watches the "source" parameter of `location.search` (the query string in the URL) in addition to `location.state` (a cross-page message hidden in the router memory). So the `CreateRelatedCommentButton` could also be written as:
+
+{% raw %}
+```jsx
+import * as React from "react";
+import Button from '@mui/material/Button';
+import { Link } from 'react-router-dom';
+
+const CreateRelatedCommentButton = ({ record }) => (
+    <Button
+        component={Link}
+        to={{
+            pathname: '/comments/create',
+            search: `?source=${JSON.stringify({ post_id: record.id })}`,
+        }}
+    >
+        Write a comment for that post
+    </Button>
+);
+```
+{% endraw %}
+
+Should you use the location `state` or the location `search`? The latter modifies the URL, so it's only necessary if you want to build cross-application links (e.g. from one admin to the other). In general, using the location `state` is a safe bet.
+
+And if you want to prefill the form with constant values, use the `defaultValues` prop on the Form tag.  
+
```

Comment:
Might be worth mentioning you can use the `CloneButton` and provide its `record` prop for that